### PR TITLE
[feature 4891] set font to Amsterdam Sans

### PR DESCRIPTION
**Pull request by SireeKoolenWijkstra**
**Context**
Per 15th of December the old font Avenir Next is no longer paid for. To prevent fines font in Sia needs het font to change to Amsterdam Sans.

**Tested by comparing local to acceptation environment** 
- happy flow voor civillians
- happy flow, filters and Signaleringen in back office
- kto-proces, not happy with result
- mijn-meldingen
- meldingenkaart

Ticket: [SIG-4891](https://datapunt.atlassian.net/browse/SIG-4891)

## Signalen

Before opening a pull request, please ensure:

- [ ] Double-check your branch is based on `main` and targets `main`
- [ ] Pull request has tests (we are going for 100% coverage!)
- [ ] Code is well-commented, linted and follows project conventions
- [ ] Committed source code is headed by the correct SPDX license expression

Be kind to code reviewers, please try to keep pull requests as small and focused as possible :)

### CODE REVIEW

**Comment by bartjkdp** [comment]
Maybe also provide a fallback for other municipalities? Not sure if everyone can use "Amsterdam Sans".

**Comment by OscarBakker** [comment]
> 

agree

**Comment by SireeKoolenWijkstra** [comment]
Inheritance is introduced by removing font setting to Amsterdam Sans in components. Only in Emailpreview was this not possible and is it kept as is.  

Fallback is introduced (set to Sans-serif). According to Jelle almost all municipalities will have their own font, so it had no importance according to the VNG.

File: src/global.css [hunk 1]
```diff
@@ -49,7 +49,10 @@ html {
 }
 
 body {
-  font: 400 1rem/1.375 Avenir Next;
+  font-family: Amsterdam Sans, sans-serif;
```
Comment (OscarBakker) [inline src/global.css:52]: check spaces

**Review by OscarBakker — APPROVED** [review]

**Pull request merged**